examples/prediction_upload/geospatial_predictions.ipynb

```diff
@@ -0,0 +1,1563 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    from PIL import Image
</pre><p>Also remove imports here, since we are already importing them at the top of the doc</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>